app/src/main/java/com/android/unio/model/association/AssociationViewModel.kt

```diff
@@ -133,6 +134,48 @@ constructor(
         { exception -> Log.e("AssociationViewModel", "Failed to update follow", exception) })
   }
 
+  /**
+   * Deletes an event from the events list of the selected association locally.
+   *
+   * @param eventId The ID of the event to be deleted.
+   */
+  fun deleteEventLocally(eventId: String) {
+    val selectedAssociation = _selectedAssociation.value
+    if (selectedAssociation != null) {
+      val eventToDelete = selectedAssociation.events.uids.find { it == eventId }
+      // if event exists ->  remove it from the events list
+      if (eventToDelete != null) {
+        selectedAssociation.events.remove(
+            eventId) // check the definition of remove to see that it does not fetch the database ;
```

Comment:
I think the comment about checking the definition was meant to be temporary

Reply:
It was for my reviewers - meaning it was for you ;) 
I'll remove it

Reply:
Update, I still think it makes it clear for the people reading it. Can I keep it ?

Reply:
Then maybe shorten it to `// does not fetch the database`, it's true that it could clarify the code a bit